components/application-gateway/internal/httpconsts/httpconsts.go

```diff
@@ -1,9 +1,17 @@
 package httpconsts
 
 const (
-	HeaderContentType   = "Content-Type"
-	HeaderAuthorization = "Authorization"
-	HeaderAccessToken   = "Access-Token"
+	HeaderContentType     = "Content-Type"
+	HeaderAuthorization   = "Authorization"
+	HeaderAccessToken     = "Access-Token"
+	HeaderCSRFToken       = "X-csrf-token"
+	HeaderCSRFTokenVal    = "fetch"
+	HeaderSetCookie       = "set-cookie"
```

Comment:
HeaderSetCookie and HeaderCookie constants not used.

Reply:
removed